libs/input-mapping/src/Table/Strategy/ABS.php

```diff
@@ -25,7 +28,7 @@ public function handleExports(array $exports, bool $preserve): array
         $jobIds = array_map(function ($export) {
             return $export[0];
         }, $exports);
-        $jobResults = $this->clientWrapper->getBasicClient()->handleAsyncTasks($jobIds);
+        $jobResults = $this->clientWrapper->getBranchClientIfAvailable()->handleAsyncTasks($jobIds);
```

Comment:
Tady bude vzdy branchovany client?

Reply:
Neslo by teda tady rovnou volat `getBranchClient()` nebo to ma jeste nejaky zadrhel? :)

Reply:
Šlo, teď už nemá https://github.com/keboola/storage-api-php-client-branch-wrapper/pull/17/files#diff-c4c109b22ff4e8fadd7042000081cd4a27c086598991ad1dcb6783f48e949a61R71 akorát bych to pak udělal někdy naráz

Reply:
A proc je tady vzdy branchovany klient plz? Nebude to vadit obyc projektum nebo "fake" dev projektum?

Reply:
Protože tam může být - ultimátním cílem je mít jednoho branchovanýho klienta ideálně všude, kromě situací, kde to nejde což je:
- něco, kde to nepodporuje storage (generateRunId)
- současný fake dev storage (getTable a spol.)

Což je ta změna oproti předhcozímu přístupu, kde jsme dávali branchovanýho klienta jen pokud to šlo (kde bylo branch id) a kde to šlo (kde to storage podporovala).

Reply:
OK